open5gs_vm/changelog.md

```diff
@@ -3,6 +3,7 @@
 ## [unreleased]
 ### Added
 - Initial version of the Open5GS VM component.
```

Comment:
We need to clarify how to work with the changelog file. Tomorrow we will review it after create the global version.